application/common/models/ResetBase.php

```diff
@@ -36,11 +37,13 @@ public static function tableName()
     public function rules()
     {
         return [
-            [['user_id', 'type', 'code', 'expires', 'created'], 'required'],
```

Comment:
I removed the default from the migration so it only exists in our extended model.